lighthouse-core/audits/network-requests.js

```diff
@@ -46,7 +46,7 @@ class NetworkRequests extends Audit {
           transferSize: record.transferSize,
           statusCode: record.statusCode,
           mimeType: record._mimeType,
-          resourceType: record._resourceType && record._resourceType._name,
+          resourceType: (record._resourceType || 'other').toLowerCase(),
```

Comment:
lowercase is confusing here, but I'm assuming this is to keep compatibility with old `ResourceType.name`...is it worth keeping it that way or should we just switch to the same capitalization as `Page.ResourceType`?

Reply:
yeah this was for compat, but we're pre 3.0 official so I'm game to remove if others think it makes sense :)

Reply:
eh. let's nuke the toLowerCase. 🔥